packages/gestalt/src/utils/positioningUtils.js

```diff
@@ -77,8 +77,10 @@ export const getTriggerRect = ({
   anchor: HTMLElement,
   positionRelativeToAnchor: boolean,
   scrollBoundaryContainerRef: ?HTMLElement,
-  // $FlowFixMe[unclear-type]
-|}): * => {
```

Comment:
This is the starting point of the changes — everything else is downstream of this change

Reply:
Love these changes, thank you Ryan.
Would be nice if we had no more `$FlowFixMe`'s in Gestalt's codebase, especially in exported code.

Reply:
Agreed. Not sure if it's something we'll be able to prioritize, but we can chip away at it with lil cleanup PRs.